examples/ecs/observers.rs

```diff
@@ -60,13 +60,13 @@ impl Mine {
     }
 }
 
-#[derive(Event, EntityEvent)]
```

Comment:
Just checking: was `EntityEvent` purposefully removed here, or was this a mistake?